test/envs/external_test.py

```diff
@@ -7,11 +7,17 @@
 ENV = os.path.join("envs", "external.yaml")
 
 
-def test_load():
+def create_addons(tmpdir):
+    tmpdir.mkdir("example")
+
+
+def test_load(tmpdir):
+    create_addons(tmpdir)
     with open(ENV) as f:
-        envfile.load(f)
+        envfile.load(f, addons_dir=str(tmpdir))
```

Comment:
Like example_test.py, we don't want to fake the addons so the test will fail if you
do a bad change in the actual env file.

This PR should have no changes in files under `envs/`.